backend/tests/apps/slack/apps_test.py

```
@@ -0,0 +1,107 @@
+            with patch.object(module, "logger") as mock_logger:
+                error_handler(test_error, test_body)
+
+            mock_logger.exception.assert_called_once_with(
+                test_error, extra={"body": test_body}
+            )
+        finally:
+            self._restore_module()
+
+    def test_log_events_creates_event_and_calls_next(self):
+        """Test log_events creates Event and calls next()."""
+        handlers, module = self._reload_with_mock_app()
```

Ruff (0.15.1)

[warning] Unpacked variable `module` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)